webapp/channels/src/components/sidebar/sidebar_category/sidebar_category_menu/sidebar_category_menu.tsx

```diff
@@ -127,9 +126,7 @@ const SidebarCategoryMenu = (props: Props) => {
         );
     }
 
-    function handleSortChannels(event: MouseEvent<HTMLLIElement> | KeyboardEvent<HTMLLIElement>, sorting: CategorySorting) {
-        event.preventDefault();
```

Comment:
🗒️ 
These defaults are not needed because Menu Items handles them at their side.